test/selectors-relative.spec.js

```diff
@@ -1,33 +1,38 @@
-const Promise = require("promise");
-const express = require("express");
-const { firefox } = require("playwright");
+const fs = require("fs");
+const process = require("process");
+const { chromium } = require("playwright");
```

Comment:
Why change the browser ?

Reply:
First because Firefox don't start on my computer.
Secondly I hop that Chrome is more used than Firefox today.
But I you really prefer to have Firefox as default I can rolling back

Reply:
I added an option for Firefox in package.json (npm run test:firefox) and I added it on the CI